examples/image-vis-type-example.ipynb

```diff
@@ -26,22 +27,13 @@
    "cell_type": "code",
    "execution_count": 1,
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "Populating the interactive namespace from numpy and matplotlib\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
-    "%pylab inline\n",
-    "\n",
     "import os\n",
-    "\n",
     "from mapboxgl.utils import img_encode\n",
-    "from mapboxgl.viz import ImageViz"
+    "from mapboxgl.viz import ImageViz\n",
+    "from numpy import mean\n",
+    "from imread import imread"
```

Comment:
@ryanbaumann `imread` comes from `matplotlib` which is already installed https://github.com/mapbox/mapboxgl-jupyter/blob/cf08d3e2fe69e866838cb495d83380d436c4f0dc/setup.py#L34 so no need to install `imread`